scripts/zones/Crawlers_Nest_[S]/npcs/Rodeupansat.lua

```diff
@@ -0,0 +1,48 @@
+-----------------------------------
+-- Area: Crawlers_Nest_[S]
+-- NPC: Rodeupansat
+-- Starts and Finishes Quest: EVIL_AT_THE_INLET
+-- !pos 129 -34 26 171
+-----------------------------------
+package.loaded["scripts/zones/Crawlers_Nest_[S]/TextIDs"] = nil
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/keyitems")
+require("scripts/globals/quests")
+require("scripts/globals/npc_util")
+require("scripts/zones/Crawlers_Nest_[S]/TextIDs")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    local Eati = player:getQuestStatus(CRYSTAL_WAR,EVIL_AT_THE_INLET)
+
+     -- Change to BRASS_RIBBON_OF_SERVICE later when Campaign has been added.
```

Comment:
now you have 5 spaces instead of 4. you need to configure whatever editor you use, nearly all code editors have a setting to make your tabkey automatically insert the correct number of spaces. Many have [editorconfig ](https://editorconfig.org/#download)support as well which can automate away most formatting woes.